eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration.php

```diff
@@ -464,4 +466,112 @@ private function addRouterSection(ArrayNodeDefinition $rootNode)
                 ->end()
             ->end();
     }
+
+    /**
+     * Define global Semantic Configuration for RichText.
+     *
+     * @param \Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition $rootNode
+     */
+    private function addRichTextSection(ArrayNodeDefinition $rootNode)
+    {
+        $ezRichTextNode = $rootNode
+            ->children()
+                ->arrayNode('ezrichtext')
+                    ->children();
+
+        $this
+            ->addCustomTagsSection($ezRichTextNode)
+            // close "ezrichtext" node opened above
+            ->end()
+                ->end()
+                    ->end();
+    }
+
+    const CUSTOM_TAG_ATTRIBUTE_TYPES = ['number', 'string', 'boolean', 'choice'];
```

Comment:
Code style: `CUSTOM_TAG_ATTRIBUTE_TYPES` should be declared on top of the class

Reply:
A little bit "too far" from code using it, but you're right, we do it at the top. Amended in 90ed221.